rustls/src/webpki/client_verifier.rs

```diff
@@ -246,14 +239,37 @@ pub struct WebPkiClientVerifier {
 }
 
 impl WebPkiClientVerifier {
-    /// Create builder to build up the `webpki` client certificate verifier configuration.
+    /// Create builder to build up the `webpki` client certificate verifier configuration using
```

Comment:
"Create [a] builder"?

Reply:
Reworded to "Create a builder for the `webpki` client ..."

Reply:
Also fixing similar instances in other types of "Create builder to build up" (in a separate commit)